include/crm/common/results.h

```diff
@@ -108,6 +108,7 @@ enum pcmk_rc_e {
     /* When adding new values, use consecutively lower numbers, update the array
      * in lib/common/results.c, and test with crm_error.
      */
+    pcmk_rc_duplicate_id        = -1033,
```

Comment:
This commit message would be API:

Reply:
Makes sense, wish I'd caught that in 73a46d1a :)
Will fix